modules/layers/src/solid-polygon-layer/solid-polygon-layer-fragment.glsl.js

```diff
@@ -24,8 +24,13 @@ export default `\
 precision highp float;
 
 varying vec4 vColor;
+varying float vEnabled;
 
 void main(void) {
+  if (vEnabled < 0.5) {
```

Comment:
How many percent of the verts are going through this loop? I.e. being rasterized just to be discarded? 2 per poly? So the perf hit on average is quite small... Maybe explain this and add an estimate about that above in the JS comments?

Reply:
1 per poly